script/deploy/healthCheck.mts

```diff
@@ -178,7 +178,7 @@ const main = defineCommand({
       const isDeployed = await checkIsDeployed(
         facet,
         deployedContracts,
-        publicClient
+        publicClient,
```

Comment:
same here...publicClient is the last parameter, there shouldnt be a comma at the end imho